.ci/run.sh

```diff
@@ -29,4 +29,12 @@ make check
 # Execute the tests under `clearcontainers/tests` repository.
 test_repo="github.com/clearcontainers/tests"
 cd "${GOPATH}/src/${test_repo}"
-.ci/run.sh
+source /etc/os-release
+
+if [ "$ID" == "ubuntu" ] && [ "$VERSION_ID" == "17.04" ]; then
```

Comment:
If we need this to work around CI issues, I think it would still be safer to extract the major version from `VERSION_INFO` and set the variable if `major_version > 16`.

Reply:
@jodh-intel this is just for debug the PR, only cri-o is failing with the new image and seems to be a random issue, in the logs I can see that sometimes the pod was not created but I am not sure. After I could capture more information I will remove the commit.

Reply:
gotcha - that's fine then ;)